code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+	casingtype = /obj/item/ammo_casing/c10mm/ap
+	projectilesound = 'sound/weapons/gunshots/gunshot_pistol.ogg'
+	environment_smash = FALSE
+	see_in_dark = 8
+
+/mob/living/simple_animal/hostile/syndicate/ranged/dsbssigma/death()
+	new /obj/effect/gibspawner/human(get_turf(src))
+	return ..()
+
+/mob/living/simple_animal/hostile/alien/larva
+	name = "alien larva"
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "larva1"
+	icon_dead = "larva1_dead"
+	layer = 2.5
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/xenomeat = 1)
+	maxHealth = 25
+	health = 25
+	retreat_distance = 4
+	minimum_distance = 1
+	density = 0
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	pass_flags = PASSTABLE | PASSMOB
+	mob_size = MOB_SIZE_SMALL
+	environment_smash = FALSE
+	status_flags = null
+	gold_core_spawnable = NO_SPAWN
+	death_sound = null
+	deathmessage = "lets out a waning high-pitched cry."
+
+/mob/living/simple_animal/hostile/alien/stationary
+	wander = FALSE
+
+/mob/living/simple_animal/hostile/alien/drone/stationary
+	wander = FALSE
+
+/mob/living/simple_animal/hostile/alien/sentinel/stationary
+	wander = FALSE
+
+/mob/living/simple_animal/hostile/alien/queen/large/stationary
+	wander = FALSE
+
+////////////////////////// TURFS //////////////////////////
+
+/turf/unsimulated/wall/fakeglass/reinforced
+	name = "reinforced plastitanium window"
+	desc = "An evil looking window of plasma and titanium. This one looks heavily reinforced."
+	icon = 'icons/obj/smooth_structures/plastitanium_window.dmi'
+	icon_state = "plastitanium_window"
+	layer = 3.2
+	level = 3
+	blocks_air = TRUE
+
+/turf/unsimulated/wall/metal/reinforced
+	name = "reinforced wall"
+	desc = "An evil wall of plasma and titanium. This one looks heavily reinforced."
+	icon = 'icons/turf/walls/plastitanium_wall.dmi'
+	icon_state = "map-shuttle_nd"
+	blocks_air = TRUE
+
+/turf/unsimulated/wall/asteroid
+	name = "dense rock"
+	desc = "Densely packed asteroid rock."
+	icon = 'icons/turf/walls.dmi'
+	icon_state = "rock"
+	blocks_air = TRUE
+
+/turf/simulated/asteroid
+	name = "loose rock"
+	desc = "Loosely packed asteroid rock. A pickaxe or drill should be able to clear it out."
+	icon = 'icons/turf/walls.dmi'
+	icon_state = "rock"
+	opacity = 1
+	layer = 4.3
+	baseturf = /turf/simulated/floor/plating/asteroid/airless
+	blocks_air = TRUE
+	density = TRUE
+	var/last_act = 0
+	var/turf/simulated/floor/plating/turf_type = /turf/simulated/floor/plating/asteroid/airless
+	var/defer_change = 0
+
+/turf/simulated/asteroid/cave
+	turf_type  = /turf/simulated/floor/plating/asteroid
+	baseturf = /turf/simulated/floor/plating/asteroid
+	defer_change = 1
+
+/turf/simulated/asteroid/attackby(obj/item/I, mob/user, params)
+	if(!user.IsAdvancedToolUser())
+		to_chat(usr, "<span class='warning'>You don't have the dexterity to do this!</span>")
+		return
+
+	if(istype(I, /obj/item/pickaxe))
```

Comment:
Can you make this into an early return like you did with the advancedToolUser one?